modules/deepspace/assets/ships/alienGeneral/aliengeneral.json

```diff
@@ -0,0 +1,71 @@
+/*
+Ship sprite from MillionthVector is licensed under a Creative Commons Attribution 4.0 International License.
+millionthvector.blogspot.id
+*/
+
+{
+  "size": 1,
+  "maxLife": 80,
+  "type": "std",
+  "price": 2000,
+  "ability": {
+    "type": "sloMo",
+    "factor": 0.3,
+    "rechargeTime": 5
+  },
+  "e1Pos": "0.11 0.43",
+  "e2Pos": "0.11 0.57",
+  "displayName": "Alien",
+  "engine": "core:techieEngine",
+  "gunSlots": [
+    {
+      "position": "0.54 0.26",
+      "isUnderneathHull": false,
+      "allowsRotation": false
+    }
+  ],
+  "rigidBody": {
+    "origin":{"x":0.93,"y":0.56},
+    "polygons":[[
+    {
+    "x":0.9399999976158142,
+    "y":0.5562500357627869},
+    {
```

Comment:
4 spaces. The formatting is off as well (end bracket on same line as variable), but I might let that slide by.